ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2318,6 +2318,12 @@ sflow:
 
 ```yaml
 snmp_server:
+  engine_ids:
+    local: < engine_name in hex >
```

Comment:
local: < engine_id in hex >, this is engine ID right?

Reply:
yes - not sure what you mean by that comment maybe removing `hex`?

Reply:
`local: < engine_id_in_hex >`